src/ixdat/techniques/ms.py

```diff
@@ -425,21 +427,44 @@ def gas_flux_calibration(
             tspan_bg (iter): Optional timespan at which the signal is at its background.
             ax (matplotlib axis): the axis on which to indicate what signal is used
                 with a thicker line. Defaults to none
+            carrier_mol (str): The name of the molecule of the carrier gas if
+                a dilute analyte is used. Calibration assumes total flux of the
+                capillary is the same as the flux of pure carrier gas. Defaults
+                to None.
+            mol_conc (float): concentration of the dilute analyte in the carrier gas
```

Comment:
I would greatly prefer `mol_conc` to be a simple mol fraction (i.e., 1 ppm corresponding to `mol_conc=1e-6`). It might be hard to remember it's in ppm when you have, say, a 1% gas mixture (which I'd think of as `mol_conc=0.01`).
Can you either (i) change `mol_conc` to a simple mol fraction, or (ii) rename it to `mol_conc_ppm` ?

Reply:
Hi Soren,
Thanks for the feedback, good point. I decided to go for `mol_conc_ppm`. I went for the unit ppm on purpose, to discourage using this function if the analyte concentration is in the percent range. Having that in the name will only help with this. :)

Reply:
Good point. I like it with the rename.